context.go

```diff
@@ -39,8 +39,8 @@ func (c *Context) Response() http.ContextResponse {
 	return NewContextResponse(c.instance, &ResponseOrigin{Ctx: c.instance})
 }
 
-func (c *Context) WithValue(key string, value any) {
-	ctx := context.WithValue(c.instance.UserContext(), ctxKey(key), value)
+func (c *Context) WithValue(key any, value any) {
+	ctx := context.WithValue(c.instance.UserContext(), key, value)
```

Comment:
Could you add some test cases with different keys? And Can the `Can we remove this?` be removed?

Reply:
done in https://github.com/goravel/fiber/pull/102/commits/9799af9b05707826c7cd304de6a893d56b704f0b